src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/HC.xaml

```diff
@@ -83,7 +83,7 @@
 
     <SolidColorBrush x:Key="ControlSolidFillColorDefaultBrush" Color="{StaticResource SystemColorButtonFaceColor}" />
 
-    <SolidColorBrush x:Key="SystemColorWindowColorBrush" Color="Transparent" />
+    <SolidColorBrush x:Key="SubtleFillColorTransparentBrush" Color="Transparent" />
```

Comment:
I don't see this resource being used anywhere. Is this part of our parity work?

Reply:
Yes. Someone may use this brush in custom style, and when they switch to HighContrast, we don't want to show errors due to missing keys.